Matrix.py

```diff
@@ -0,0 +1,175 @@
+""" in this program Matrix is a tuple of tuples and the internal tuples must be as long as the
+external list is.data is the Matrix (tuple) received from the user, and this class has add, sub,
+ mul, scalar_mul and division by scalar methods and non- mathemathic methods
+ as repr, str, hash and eq."""
+
+
+class Matrix:
+    """A matrices calculator for basic math actions
+        Attributes:
+            data(tuple of tuples): a Matrix- each line is a tuple in the major tuple """
```

Comment:
This isn't how you write class doc, read about code styling in the repo

Reply:
ok